webapp/components/delete_post_modal.jsx

```diff
@@ -54,8 +54,8 @@ export default class DeletePostModal extends React.Component {
             this.state.post.channel_id,
             this.state.post.id,
             () => {
-                PostStore.deletePost(this.state.post);
-
+                PostStore.removePost(this.state.post);
```

Comment:
This should really be an action. You can see examples here: https://github.com/mattermost/platform/blob/master/webapp/actions/post_actions.jsx#L88

You can find more info on our architecture (flux) here: https://www.youtube.com/watch?v=8dB2_29TtNo

We would really appreciate you turning this into an action.